oat-sa/tao-extension-release
# [TAO-10603] TAO Release tool - Calculate the version from conventional commits

Related to : https://oat-sa.atlassian.net/browse/TAO-10603
 
Calculate the version from conventional commits. Update version during release 
  
#### How to test
- add some changes to playground extension https://github.com/oat-sa/extension-tao-foobar
- use conventional commits messages when publishing changes https://www.conventionalcommits.org/en/v1.0.0/
- try to release new version of extension
- check that version properly calculated
- check that version update during release process

ToDo : 
 - [x] Merge and release https://github.com/oat-sa/conventional-changelog-tao/pull/1 
 - [x] Update the package.json and sync the lock file
